src/pip/_internal/utils/misc.py

```diff
@@ -226,6 +227,18 @@ def is_svn_page(html):
             re.search(r'Powered by (?:<a[^>]*?>)?Subversion', html, re.I))
 
 
+def is_within_directory(directory, target):
+    # type: (str, str) -> bool
+    """
+    Return true if the absolute path of target is within the directory
+    """
+    abs_target = os.path.abspath(target)
+    abs_directory = os.path.abspath(directory)
```

Comment:
This is minor, but it would help for keeping these two variables straight if `abs_directory` and `abs_target` can be defined in the same order as they appear in the argument list.